### ISSUE TITLE
Traefik: different port for specific sub domain

### ISSUE THREAD

**Issue by ElVit** [alex3305/home-assistant-addons#45]
Hi,
I am using your addon for a quite long time in combination with Cloudflare and I’m very happy with it. 
I am still not understanding everything how Traefik works, so I was wondering if it would be possible to expose some specific sub domains over a different port than 443. 
For example a mail server would need port 993 (IMAP) and port 587 (SMTP) but also with SSL. 

Is this possible with your addon? 
Also if not could you please tell me how the static and dynamic config should look like for my specific use case?
Thanks!

**Comment by alex3305** [comment]
Hi @ElVit,

I always appreciate your feedback and I really like that you enjoy using this add-on. I thought about this for a couple of days. But to be honest I don't think this add-on should have functionality to also proxy mail servers. Let me explain why.

First of all, Home Assistant add-ons are expected to use the Home Assistant Ingress (you know, the **View Web UI** button). Of course explicit port forwarding is allowed, but here lies our first problem. If I'm going to add, let's say, ports 993 and 587. Than why shouldn't I also add 995 for POP access. Or perhaps 25 for regular SMTP access? Or any other services that could be proxied through Traefik. 

That brings me to my second point. I think this add-on was designed to forward Home Assistant. Because if you want to do any additional forwarding, you should probably roll your own solution. For instance within Docker on unRAID, as I'm currently doing.

If you want to create your own solution, I suggest you check out the configuration files within your own Traefik add-on. These are located at:

* `/etc/traefik/traefik.yaml`
* `/config/traefik/*.yaml` (your configured dynamic configuration)

For mail servers, I suggest you take a look at [something like this](https://docker-mailserver.github.io/docker-mailserver/edge/examples/tutorials/mailserver-behind-proxy/).

Good luck and I hope you will succeed in what you want! 🚀

**Comment by ElVit** [comment]
Hi @alex3305,
I know its a while ago since my last comment here.
Thank you for your idea with Unraid. Finally I have also switched to Unraid and I like it 😃.
Also Traefik is woring very well here. 
I followed the manual from [ibracorp](https://docs.ibracorp.io/traefik).

I have one more question regarding Unraid and Home Assistant:
You have also created the addon "Bitwarden secrets for Home Assistant" which is very usefull for me.
Is it possible to run this addon also in Unraid?

Currently I run Home Assistant in a VM, but I want to switch to a Docker Container.
So all of my Addons have to run in Unraid Docker.
